### ISSUE TITLE
aireplay-ng frag attack 384 byte keystream corrupted

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#110]
**Reported by darkAudax on 16 Feb 2007 20:30 UTC**

This was tested with aircrack-ng/aircrack-ng@. I believe prior versions are affected as well.

When trying to use the 384 byte output file with packetforge-ng, you get an index out of range error message.

```
packetforge-ng -0 -a 00:14:6C:7E:40:80 -h 00:11:10:54:4F:5B -k 255.255.255.255 -l 255.255.255.255 -y fragment-0215-195927.xor -w arp.cap                 
Are you really sure that this is a valid keystream? Because the index is out of range (0-3): 40
Wrote packet to: arp.cap
```

**Comment by aircrack-ng** [comment]
**Comment by hirte on 16 Feb 2007 20:30 UTC**

Fixed in aircrack-ng/aircrack-ng@f8a9f070285845f8ac127653548d6d8c88604727

**Comment by aircrack-ng** [comment]
**Modified by misterx on 16 Feb 2007 20:30 UTC**
